server/images.go

```diff
@@ -414,14 +415,14 @@ func CreateModel(ctx context.Context, name model.Name, modelFileDir, quantizatio
 				}
 				defer blob.Close()
 
-				baseLayers, err = parseFromFile(ctx, blob, digest, fn)
+				baseLayers, err = parseFromFile(ctx, command, baseLayers, blob, digest, fn)
```

Comment:
Keep a copy of, and pass in, `ggml.KV()`

```suggestion
				baseLayers, err = parseFromFile(ctx, modelKV, blob, digest, fn)
```

Reply:
What's your thinking on this? I had thought about doing it that way, but it felt odd vs. just passing in the other layers.

Reply:
the only relevant property is model KV so passing in other layers, which may have layers for parameters, templates, messages, etc, is unnecessary

Reply:
The problem is we may need to reference something in another layer, like potentially if there are other adapters.

Reply:
I switched it and it's reasonably easy enough to plumb this back in if we need to.